### ISSUE TITLE
Fails to detect item 

### ISSUE THREAD

**Issue by rutexd** [PrismarineJS/MineflayerArmorManager#53]
Spam with error on item pickup, even without any related code to armor manager at all.

Mineflayer 4.8.1
mineflayer-armor-manager: 1.4.2
Minecraft: 1.19.3/java
Nodejs: 18.14.0
```
Failed to retrieve block id, probably exp bottle TypeError: Cannot read properties of null (reading 'findItemOrBlockById')
    at Object.isArmor (C:\dev\NodeMinecraftProtocol\ArtIdiot\node_modules\mineflayer-armor-manager\dist\lib\isArmor.js:7:30)
    at EventEmitter.onPlayerCollect (C:\dev\NodeMinecraftProtocol\ArtIdiot\node_modules\mineflayer-armor-manager\dist\index.js:42:37)
    at EventEmitter.emit (node:events:525:35)
    at EventEmitter.emit (node:domain:489:12)
    at Client.<anonymous> (C:\dev\NodeMinecraftProtocol\ArtIdiot\node_modules\mineflayer\lib\plugins\entities.js:170:9)
    at Client.emit (node:events:513:28)
    at Client.emit (node:domain:489:12)
    at FullPacketParser.<anonymous> (C:\dev\NodeMinecraftProtocol\ArtIdiot\node_modules\mineflayer\node_modules\minecraft-protocol\src\client.js:91:12)
    at FullPacketParser.emit (node:events:513:28)
    at FullPacketParser.emit (node:domain:489:12)
```

**Comment by siddvish21** [comment]
Same issue.

**Comment by ExoOnix** [comment]
I am also having the same issue currently.

**Comment by G07cha** [comment]
Should be resolved in v2.0.0, let me know if it's not the case.
